bookwyrm/models/fields.py

```diff
@@ -423,6 +424,20 @@ def formfield(self, **kwargs):
             }
         )
 
+    # pylint: disable=no-self-use
+    def get_absolute_url(self, value):
+        """returns an absolute URL for the image"""
+        name = getattr(value, "name")
+        if name is None:
```

Comment:
I think this is too limited a check - it looks like `name` may be an empty string. Maybe `if not name:`?